fv3core/stencils/map_single.py

```diff
@@ -34,6 +34,8 @@ def lagrangian_contributions(
     dp1: FloatField,
     q2_adds: FloatFieldIJ,
 ):
+    with computation(FORWARD), interval(0, 1):
+        q2_adds=0
```

Comment:
I think this would be clearer if you move this down before line 80 to where it is used. Otherwise it looks like q2_adds could be used in the next computation before the adds at the end of the stencil.